### PR TITLE
feat: Strings Exporter Settings API

### PR DESCRIPTION
**Pull request by kisharnath**

### CODE REVIEW

**Comment by kisharnath** [comment]
please check now

File: src/projectsGroups/index.ts [hunk 1]
```diff
@@ -272,6 +272,70 @@ export class ProjectsGroups extends CrowdinApi {
         const url = `${this.url}/projects/${projectId}/file-format-settings/${fileFormatSettingsId}`;
         return this.patch(url, request, this.defaultConfig());
     }
+
+    /**
+     * @param projectId project identifier
+     * @param options optional parameters for the request
+     * @see https://developer.crowdin.com/api/v2/#operation/api.projects.strings-exporter-settings.getMany
+     */
+    listProjectStringExporterSettings(
```
Comment (andrii-bodnar) [inline src/projectsGroups/index.ts:281]:
```suggestion
    listProjectStringsExporterSettings(
```


make it plural here and below (ProjectString**s**ExporterSettings)

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

File: src/projectsGroups/index.ts (hunk 1 above)
Comment (andrii-bodnar) [reply src/projectsGroups/index.ts]: @kisharnath including the other methods, types and interfaces

File: src/projectsGroups/index.ts [hunk 2]
```diff
@@ -272,6 +272,70 @@ export class ProjectsGroups extends CrowdinApi {
...
+    listProjectStringsExporterSettings(
+        projectId: number,
+        options?: PaginationOptions,
+    ): Promise<ResponseList<ProjectsGroupsModel.ProjectStringsExporterSettings>> {
+        const url = `${this.url}/projects/${projectId}/strings-exporter-settings`;
+        return this.getList(url, options?.limit, options?.offset);
+    }
+
+    /**
+     * @param projectId
```
Comment (andrii-bodnar) [inline src/projectsGroups/index.ts:290]:
```suggestion
     * @param projectId project identifier
```

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

**Review by andrii-bodnar — APPROVED** [review]
@kisharnath thank you!

**Pull request merged**